addons/website_livechat/views/website_livechat.xml

```diff
@@ -234,11 +228,7 @@
 
         <!-- User Navbar -->
         <template id="user_navbar_inherit_website_livechat" inherit_id="website.user_navbar">
-            <xpath expr="//div[@id='o_new_content_menu_choices']//div[@name='module_website_livechat']" position="attributes">
-                <attribute name="name"/>
-                <attribute name="t-att-data-module-id"/>
-                <attribute name="t-att-data-module-shortdesc"/>
-                <attribute name="groups">im_livechat.im_livechat_group_manager</attribute>
+            <xpath expr="//div[@id='o_new_content_menu_choices']//div[@name='module_website_livechat']" position="replace">
```

Comment:
Can you do the same for website_forum too please.

It follow the same logic that here... 
Once installed, no reason (or little) to create another one so we can remove the button
(@sbuhl spec)